compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToUKernels.cpp

```diff
@@ -232,8 +232,8 @@ void AMDAIELowerToUKernelsPass::runOnOperation() {
                                                             pathToUkernels);
   patterns.insert<LowerToUKernelPattern<linalg::MatmulOp>>(context, allTargets,
                                                            pathToUkernels);
-  patterns.insert<LowerToUKernelPattern<linalg::FillOp>>(context, allTargets,
-                                                         pathToUkernels);
```

Comment:
@Abhishek-Varma is this supported on mlir-aie yet? Doesn't seem to work on my branch.